google-cloud-bigtable/pom.xml

```diff
@@ -60,11 +60,11 @@
   <dependencies>
     <!-- NOTE: Dependencies are organized into two groups, production and test.
          Within a group, dependencies are sorted by (groupId, artifactId) -->
-
     <dependency>
       <groupId>com.google.cloud</groupId>
       <artifactId>google-cloud-bigtable-stats</artifactId>
-      <!-- Exclude all the shaded transitive dependencies -->
+      <!-- Exclude all the shaded transitive dependencies. After mvn clean install is run,
```

Comment:
Exclude all dependencies that have been shaded. This is to workaround maven's immutable project structure:
after shading, the maven-shade-plugin tries to remove shaded deps, but it can't since the project structure is immutable. So we have to manually exclude the shaded transitive deps manually